src/main/java/com/three_tech_solutions/slot_app/services/implementations/StudentServiceImpl.java

```diff
@@ -169,7 +168,12 @@ public void validateIfDniExists(String dni) {
             throw new ResponseStatusException(HttpStatus.NOT_FOUND, "El DNI no está registrado");
         }
     }
-
+    private Integer calculateStudentAge(LocalDate birthday) {
+        if (birthday == null) {
+            return null;
+        }
+        return Period.between(birthday, LocalDate.now()).getYears();
+    }
```

Comment:
En teoria el `birthday` no podria ser null, ya que es un dato obligatorio cuando damos de alta un estudiante.
Quitando eso, esto lo podrias refactorizar asi aprovechando la API de `Optional` que nos permite "trabajar" los datos que podrian ser nulos


```suggestion
    private Integer calculateStudentAge(LocalDate birthday) {
        return Optional.ofNullable(birthday)
                .map(date -> Period.between(date, LocalDate.now()).getYears())
                .orElse(null);
    }
```